### PR TITLE
Add flake8 to the Travis build

### PR DESCRIPTION
**Pull request by eliasdorneles**
The motivation is to avoid bikeshedding on the PRs about PEP-8 stuff. :)

I put a relaxed line limit conforming to the current code, seemed good enough.

Does this look good?

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -10,7 +10,8 @@ deps =
     mock
     pytest
     pytest-cov
```
Comment (jdemaeyer) [inline tox.ini]: should `flake8` be listed as dependency here?

Comment (eliasdorneles) [inline tox.ini]: it is. :)
the diff is confusing.

File: tox.ini [hunk 2]
```diff
@@ -10,7 +10,8 @@ deps =
...
-commands =
+    flake8
+commands = flake8
```
Comment (jdemaeyer) [inline tox.ini]: Speaking of bikeshredding, could you add a newline before `flake8` here? ;) Almost read over it if it wasn't for the diff highlighting

Comment (eliasdorneles) [inline tox.ini]: I tried that, but I'm not sure why, the build doesn't fail when `flake8` fails if I move it to the next line.

Comment (eliasdorneles) [inline tox.ini]: Hm, I can't reproduce it anymore so it was probably supertitious of me. Lemme do the change. :)

File: tox.ini (hunk 1 above)
Comment (eliasdorneles) [inline tox.ini]: added a newline so the diff isn't confusing anymore :)

**Comment by jdemaeyer** [comment]
Thanks Elias :) could you rebase?

**Comment by eliasdorneles** [comment]
The build `freeze` seems to have gotten stuck.
In any case, this seems to be pretty safe (passed all other builds), right @jdemaeyer ?

**Comment by jdemaeyer** [comment]
yup, thanks!

**Pull request merged**